graylog2-server/src/main/java/org/graylog/events/indices/MoreIndices.java

```diff
@@ -16,50 +16,36 @@
  */
 package org.graylog.events.indices;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import io.searchbox.client.JestClient;
-import io.searchbox.core.Bulk;
-import io.searchbox.core.BulkResult;
-import io.searchbox.core.Index;
 import org.graylog.events.event.Event;
-import org.graylog.events.event.EventDto;
 import org.graylog.events.event.EventWithContext;
-import org.graylog2.indexer.IndexMapping;
-import org.graylog2.jackson.TypeReferences;
 import org.graylog2.plugin.database.Persisted;
 import org.graylog2.streams.StreamService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import java.io.IOException;
+import java.util.AbstractMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import static java.util.Objects.requireNonNull;
-import static org.graylog2.plugin.Tools.buildElasticSearchTimeFormat;
-import static org.joda.time.DateTimeZone.UTC;
-
 /**
  * This class contains indices helper for the events system.
  */
 @Singleton
 public class MoreIndices {
```

Comment:
Wouldn't it make sense to rename this to `EventIndices` while we're at it? I find `MoreIndices` too vague for it's very specific purpose. `EventWriter` or `EventIndexer` would be even more specific. Same would go for `MoreIndicesAdapter` and `MoreIndicesAdapterES6`.

The method name `bulkIndex` on the other hand is unnecessarily specific. `write` would be a fine alternative imo.